README.md

```diff
@@ -49,7 +49,27 @@ module.exports = function(app) {
 // etc...
 ```
 
-### Model Definitions
+### Variable Declarations
+
+Keep variable names in alphabetical order
+
+*Why?:* Easier to read and easier to find what you are looking for.

```

Comment:
Makes sense. Having a logical flow like this could add efficiency when looking through a lot of code. I don't see any reason why we shouldn't - it doesn't really add extra time, just additional consideration.